### DISCUSSION TITLE
Outer glow effect

### DISCUSSION THREAD

**Discussion by defunky** [dlemstra/Magick.NET#1177] [Help]
Whats the best way to achieve an outer glow border around a transparent image similar to this:
![image](https://user-images.githubusercontent.com/17749501/163720564-bbc232ce-97a9-49f2-9adb-abad9cacc880.png)

Would it be to copy the base image, resize it and blur it? or is there a better method of achieving such an effect?

**Comment by dlemstra** [comment]
It might be better to first ask that question here: https://github.com/ImageMagick/ImageMagick/discussions. There are more experts on how to do that there. I can help you if you need help translating their command line commands back to code.

**Reply by defunky** [reply]
Thanks for the reply, I would really appreciate it if you could help me convert it:
```
convert red_rect.png \
\( -clone 0 -alpha extract \) \
\( -clone 1 -fill white -draw "color 110,175 floodfill" -alpha off \
-blur 0x7 -level 0x50% \
-fill black -draw "color 110,175 floodfill" -alpha off \) \
\( -clone 2,1 -compose plus -composite \) \
-delete 1,2 \
-alpha off -compose over -compose copy_opacity -composite \
red_rect3.png
```

```
  convert red_rect.png \
          \( +clone -background "#f7f6e4" -shadow 100x3+0+0 -channel A -level 0,50% +channel \) +swap \
          +repage -gravity center -composite   shadow_outline_darker.png
```

**Reply by dlemstra** [reply]
What have you tried yourself and where are you stuck?
